redisai/model.go

```diff
@@ -1,6 +1,7 @@
 package redisai
 
 import (
+	"github.com/RedisAI/redisai-go/redisai/implementations"
```

Comment:
wanted to avoid circular dependencies. please see the next comments

Reply:
Thanks. My mistake.